### DISCUSSION TITLE
Is it possible to install adguard to dune hd tv 101?

### DISCUSSION THREAD

**Discussion by alptolga** [AdguardTeam/AdGuardHome#3192] [Q&A]
I installed optware to dune hd tv 101. I want to use it with adhuard. Is it possible? Here is the details of dune?

 Processor: Sigma Designs 8670/8671 – The very latest high powered solution from Sigma Designs
• RAM: 256 MB
• Flash memory: 128 MB, expandable with USB flash drive

tangox[~]# uname -r
2.6.29.6-18-sigma

tangox[~]# cat /proc/cpuinfo
system type             : Sigma Designs TangoX
processor               : 0
cpu model               : MIPS 24Kc V7.12  FPU V0.0
BogoMIPS                : 468.99
wait instruction        : yes
microsecond timers      : yes
tlb_entries             : 32
extra interrupt vector  : yes
hardware watchpoint     : yes, count: 4, address/irw mask: [0x0ffc, 0x0ffc, 0x0ffb, 0x0ffb]
ASEs implemented        : mips16
shadow register sets    : 1
core                    : 0
VCED exceptions         : not available
VCEI exceptions         : not available

SMP8XXX Chip ID         : 8671
SMP8XXX Rev ID          : 2
System bus frequency    : 351000000 Hz
CPU frequency           : 702000000 Hz
DSP frequency           : 351000000 Hz

**Comment by ainar-g** [comment]
Hello.  We do provide MIPS on Linux support, but such devices often have their own peculiarities.

Also, Go is planning to drop support for Linux kernels below 3.10 in about a year or so, so future versions of AGH may not work correctly on the 2.6.29 kernel.

**Reply by alptolga** [reply]
thanks for your answer.
